custom/icds_reports/static/icds_reports/js/spec/adult-weight-scale.directive.spec.js

```diff
@@ -75,10 +75,9 @@ describe('Adult Weight Scale Directive', function () {
 
     it('tests template popup', function () {
         var result = controller.templatePopup({properties: {name: 'test'}}, {in_month: 5, all: 10});
-        assert.equal(result, '<div class="hoverinfo" style="max-width: 200px !important;">' +
-            '<p>test</p>'
-            + '<div>Total number of AWCs with a weighing scale for mother and child: <strong>5</strong></div>'
-            + '<div>% of AWCs with a weighing scale for mother and child: <strong>50.00%</strong></div>');
+        assert.equal(result, '<div class="hoverinfo" style="max-width: 200px !important;"><p>test</p>' +
+            '<div>Total number of AWCs that reported having a weighing scale for mother and child: <strong>5</strong>' +
+            '</div><div>% of AWCs that reported having a weighing scale for mother and child: <strong>50.00%</strong></div>');
```

Comment:
No big deal, but your life might get better if you also define some of this text in a `const` type of file so that you can maintain it in a single place.